.github/workflows/nightly.yml

```diff
@@ -0,0 +1,125 @@
+name: Nightly
+
+on:
+  schedule:
+    - cron: 0 0 * * *
+
+
+jobs:
+  generate-proving-key:
```

Comment:
small nit: would call it "generate-proving-keys" as it's > 1 key

Reply:
reflected in 20676ac36afb18c8d763b05e6d680aec52f0c518